webpack/assets/javascripts/react_app/components/TemplateGenerator/index.js

```diff
@@ -0,0 +1,30 @@
+import { pick } from 'lodash';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import reducer from './TemplateGeneratorReducer';
+import * as templateActions from './TemplateGeneratorActions';
+import TemplateGenerator from './TemplateGenerator';
+
+export const actions = templateActions;
+
+// export reducers
+export const reducers = { templates: reducer };
+
+// map state to props
+const mapStateToProps = ({ templates }) =>
+  pick(templates, [
```

Comment:
could you use selectors ? 
afaik `pick` will create a new object, so even if attributes haven't been changed, a re-render will occur

Reply:
I haven't realized that about `pick`. I have created selectors, they are simple ones, but you are right, feels better.